src/settings/MappingProfiles/MappingProfilesForm.js

```diff
@@ -252,6 +257,18 @@ export const MappingProfilesFormComponent = ({
     }
   };
 
+  const handleFOLIORecordTypeChange = e => {
+    const value = e.target.value;
+
+    setFolioRecordType(value);
+
+    if (value === MARC_TYPES.MARC_AUTHORITY) {
```

Comment:
Do we still need to call `setFieldMappingsForMARC` even after you added `marcMappingOption: 'UPDATE'` to MARC authority initial details?